docs/basics/k8s_configure.md

```diff
@@ -1,41 +1,40 @@
-# Configure Flex under Kubernetes
+# Configure Flex in Kubernetes
 
-If you're running under Kubernetes we recommend you use the official container images from NewRelic.
+## New Relic container images
 
-There are 3 container images you can use depending on you use-case. 
-- If you plan on using only Flex to monitor service in Kubernetes you can use the container image [newrelic/infrastructure](https://hub.docker.com/r/newrelic/infrastructure).
-  
-  This image only contains the Infrastructure agent, and the Docker and Flex integrations. You will not be able to do service discovery and use other NewRelic integrations.
-- If you plan on using Flex and perhaps other NewRelic integration you can use the container image [newrelic/infrastructure-bundle](https://hub.docker.com/r/newrelic/infrastructure-bundle)
+If you're running services in Kubernetes we recommend you use the official container images from New Relic.
 
-   This image builds on top on the [newrelic/infrastructure](https://hub.docker.com/r/newrelic/infrastructure) image and adds all the other NewRelic support integrations. See this [page](https://docs.newrelic.com/docs/integrations/kubernetes-integration/link-apps-services/monitor-services-running-kubernetes) for more information.
+There are three container images you can use depending on how you plan on using Flex: 
 
-- If you also want to monitor your Kubernetes cluster you can use the container image [newrelic/infrastructure-k8s](https://hub.docker.com/r/newrelic/infrastructure-k8s).
-  
-  This image builds on top of [newrelic/infrastructure-bundle](https://hub.docker.com/r/newrelic/infrastructure-bundle) and add the Kubernetes integration. See this [page](https://docs.newrelic.com/docs/integrations/kubernetes-integration/get-started/introduction-kubernetes-integration) for more information.
+- To run Flex only to monitor services running in Kubernetes, use the [newrelic/infrastructure](https://hub.docker.com/r/newrelic/infrastructure) container image. This image only contains the Infrastructure agent, and the Docker and Flex integrations. You will not be able to perform service discovery or use other New Relic integrations.
+- To run Flex alongside other New Relic integrations, use the [newrelic/infrastructure-bundle](https://hub.docker.com/r/newrelic/infrastructure-bundle) container image. This adds all the other For more information, see [New Relic integrations](https://docs.newrelic.com/docs/integrations/kubernetes-integration/link-apps-services/monitor-services-running-kubernetes).
```

Comment:
we have a broken sentence here:
.... This adds all the other For more information....